source/sod/install.rst

```diff
@@ -0,0 +1,50 @@
+安装
+====
+
+1.  确认 Java 运行环境已安装
+
+    在终端中键入 ``java -version``，若显示版本信息，则表示 Java 运行环境已安装::
+
+        $ java -version
+        openjdk version "1.8.0_161"
+        OpenJDK Runtime Environment (build 1.8.0_161-b14)
+        OpenJDK 64-Bit Server VM (build 25.161-b14, mixed mode)
+
+    否则需要先安装 Java 运行环境::
+
+        # CentOS/RHEL
+        $ sudo yum install java
+
+        # Fedora
+        $ sudo dnf install java-latest-openjdk
+
+        # Ubuntu/Debian
+        $ sudo apt update
+        $ sudo apt install default-jdk
+
+        # macOS homebrew 用户
+        $ brew install openjdk
```

Comment:
Copy from https://seismo-learn.org/software/taup/install/. I feel that we should add a new section "how to install java" in the appendix so that we can link to that section in both TauP and SOD.

Reply:
yes, same is true for other common dependences